### ISSUE TITLE
Tag sessions with the capturing interface

### ISSUE THREAD

**Issue by copeland3300** [arkime/arkime#1638]
**Is your feature request related to a problem? Please describe.**

No problem, but it would be useful if we could easily filter on the capturing interface.

**Describe the solution you'd like**
I don't know if this would be best as just a tag, as a discrete column in the UI, or modifying the node name.

**Describe alternatives you've considered**
N/A

**Additional context**
In my experimentation, I was able to make this work by creating an instance of each of the following files for each interface I was monitoring:
 - /data/moloch/etc/config.ini
 - /data/moloch/bin/moloch_config_interfaces.sh
 - /etc/systemd/system/molochcapture.service

The specific config.ini and moloch_config_interfaces.sh each was changed to only reference an individual interface.

I also made discrete pcap directories for each interface and specified those in the config.ini files.

The molochcapture.service files I created were specified with `-n ORIGINALNODENAE-INTERFACENAME`, `-t INTERFACENAME`, a unique log file name, and finally the unique config.ini file for `ExecStart`, and the unique moloch_config_interfaces.sh for `ExecStartPre`. I specified a unique node name, otherwise I would get errors from elasticsearch referring to the stats index, specifically looking like this:
```
{"error":{"root_cause":[{"type":"version_conflict_engine_exception","reason":"[molochVM]: version conflict, current version [22070] is higher or equal to the one provided [22070]","index_uuid":"NMAYhwULQvahWDcaYKhl1A","shard":"0","index":"stats_v4"}],"type":"version_conflict_engine_exception","reason":"[molochVM]: version conflict, current version [22070] is higher or equal to the one provided [22070]","index_uuid":"NMAYhwULQvahWDcaYKhl1A","shard":"0","index":"stats_v4"},"status":409}
```

I'd guess this is due to having two moloch_capture processes running on the same VM at the same time.

I added the tag since I wasn't sure which would work best, so I figured I'd try with a tag as well.

**Comment by copeland3300** [comment]
Generally that seems to be working and I'm seeing both the unique node name, which includes the interface name, as well as the interface name as a tag

**Comment by copeland3300** [comment]
I'm now realizing this is probably what the `[node1]` and `[node2]` sections are at the bottom of the config.ini file...

**Comment by awick** [comment]
This is already mostly possible if you are ok using the tags field:
If using 1 interface per capture just either use the `-t interfacename` option or the `--op tags=interfacename` to capture
If using multple interfaces per capture then look at the https://arkime.com/settings#interfaceops setting

If you wanted a new field called "interface" that would have to be added using https://arkime.com/settings#custom-fields, and then you would use `--op interface=interfacename` or again the interfaceops setting.

> I'd guess this is due to having two moloch_capture processes running on the same VM at the same time.

yes node name should be unique. Instead of running multiple, you should just have capture listen to 2 interfaces

> I'm now realizing this is probably what the [node1] and [node2] sections are at the bottom of the config.ini file...

not sure what you mean, but order in the file shouldn't matter. Just make sure settings you are adding actually end up in the right section.

**Comment by copeland3300** [comment]
It seems i missed [https://arkime.com/faq#how-do-i-monitor-multiple-interfaces](url)

Most of the info there is what I ended up doing, but I did differ by adding the tag and having two config files.

One question on the existing instructions: When the two different systemd services start, each one would run the `moloch_config_interfaces.sh` script. If the single script makes changes to all of the interfaces, wouldn't that potentially cause issues with other interfaces that are already capturing? For example if you're capturing on eth1 and eth2, and restart the eth2 process, since both eth1 and eth2 are referenced in the common `moloch_config_interfaces.sh`, would that impact eth1?

> not sure what you mean, but order in the file shouldn't matter. Just make sure settings you are adding actually end up in the right section.

To clear up what I meant: I was mixing up a few things, and didn't realize the use of creating unique sections in the `config.ini` file.
